keyboards/getta25/keymaps/default/keymap.c

```diff
@@ -0,0 +1,68 @@
+#include QMK_KEYBOARD_H
+
+#ifdef RGBLIGHT_ENABLE
+//Following line allows macro to read current RGB settings
+extern rgblight_config_t rgblight_config;
+#endif
+
+extern uint8_t is_master;
```

Comment:
This block isn't needed. Especially the "is_master", as this is for split keyboards.

Reply:
I deleted that code block.